src/frontPage/frontPage.tsx

```diff
@@ -68,33 +69,22 @@ const FrontPage = (): JSX.Element => {
             )}
             bottomText={t(
               'frontPage.otherCompetitionsAndSearches.explanation',
-              'Description lorem ipsum dolor sit amet et cetera et cetera.'
+              'Other on going area searches and competitions.'
```

Comment:
* The translation file has "ongoing" spelled properly, but it's still separated here.
* The Finnish labels don't end in a period and I think these work better that way, so I'd suggest dropping the periods from the English texts as well.